bindings/rust/s2n-tls-hyper/Cargo.toml

```diff
@@ -10,15 +10,22 @@ license = "Apache-2.0"
 publish = false
 
 [features]
-default = []
+default = ["http1"]
```

Comment:
Should we be defaulting to "http2" instead?

Reply:
Hmm yeah maybe. And then I guess we wouldn't even need an API to request HTTP/2. We could just always include it in the ALPN to try to negotiate it. I'll try implementing it that way.